admin/class-boldgrid-backup-admin-cron-log.php

```diff
@@ -146,9 +197,23 @@ public function get_markup() {
 		// If we're getting the markup, the user is looking at it. Mark it as read.
 		$this->set_as_read();
 
+		$jobs = $this->get_jobs();
+
 		$log = $this->get_log();
 
-		$markup = '<h2>' . __( 'Latest Cron Notices', 'boldgrid-backup' ) . '</h2>
+		$markup = sprintf(
+			'<h2>%1$s</h2>
+			<p>%2$s %3$s</p>
+			%4$s
+			</tbody>
+			</table>',
+			__( 'Scheduled Jobs', 'boldgrid-backup' ),
+			__( 'If there are any jobs scheduled to run, they will be displayed below.', 'boldgrid-backup' ),
+			__( 'Successfully completed jobs will be removed from the list the next time the cron runs.', 'boldgrid-backup' ),
+			wp_kses_post( $jobs ),
```

Comment:
`$jobs` may be a `<p>` with a `style` attribute.

```php
// Return a message if there are no jobs.
if ( empty( $jobs ) ) {
	return '<p style="font-style:italic">' . esc_html__( 'No jobs scheduled.', 'boldgrid-backup' ) . '</p>';
}
```

Will `wp_kses_post` strip either the tag or the style attribute?

Reply:
No. `wp_kses_post` is designed to allow just about anything that is normally allowed inside a post's `post_content` field. Style tags will only be stripped if they are empty. The screenshots